libsql_test.go

```diff
@@ -519,6 +520,7 @@ func TestEncryption(tt *testing.T) {
 	t.FatalOnError(err)
 	db := sql.OpenDB(connector)
 	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+	defer cancel()
```

Comment:
Will this run if tests fails with fatal?

Reply:
Yes
> execute regardless of whether completion is due to normal execution or panic.